cookbook/choose_protocol.ipynb

```diff
@@ -4,7 +4,6 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.            off_toolkit_backend='ambertools',  # Toolkit to use for partial charge assignment - ambertools
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>